integration/command.go

```diff
@@ -312,6 +318,11 @@ spec:
     - %s
 EOF
 `, namespace, cmd)
+
+	return &command{
+		name: "Run test-pod",
```

Comment:
What about making this a parameter and so keeping the old names as argument?

Reply:
I checked the old names and most of them are generic descriptions that IMO don't add any useful information. By the way, we're not using command.name anywhere...

Reply:
Isn't it printed when running with `-test.v`?
If not, then your modification is fine.

Reply:
It's not printed anywhere. We're just assigning it but not printing.

Reply:
OK, so better to remove it then.